From ec93c465945b75b5531790aa1ab91246b5600a08 Mon Sep 17 00:00:00 2001
From: afl1 <afl2001@gmail.com>
Date: Tue, 19 Mar 2019 09:35:10 +0100
Subject: [PATCH] AML: remove setting of freerun_mode 3.

AML Vendor kernel doesn't have the freerun_mode in amlvideodri anymore
and it will default to mode 3 behaviour now.
---
 xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp | 2 --
 xbmc/utils/AMLUtils.cpp                             | 5 -----
 2 files changed, 7 deletions(-)

diff --git a/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp b/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp
index ae2137728b5..d29453d0f00 100644
--- a/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp
+++ b/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp
@@ -1783,8 +1783,6 @@ bool CAMLCodec::OpenAmlVideo(const CDVDStreamInfo &hints)
   m_defaultVfmMap = GetVfmMap("default");
   SetVfmMap("default", "decoder ppmgr deinterlace amlvideo amvideo");
 
-  SysfsUtils::SetInt("/sys/module/amlvideodri/parameters/freerun_mode", 3);
-
   return true;
 }
 
diff --git a/xbmc/utils/AMLUtils.cpp b/xbmc/utils/AMLUtils.cpp
index 1a729eb50c1..a03b58b5783 100644
--- a/xbmc/utils/AMLUtils.cpp
+++ b/xbmc/utils/AMLUtils.cpp
@@ -87,11 +87,6 @@ bool aml_permissions()
       CLog::Log(LOGERROR, "AML: no rw on /dev/video10");
       permissions_ok = 0;
     }
-    if (!SysfsUtils::HasRW("/sys/module/amlvideodri/parameters/freerun_mode"))
-    {
-      CLog::Log(LOGERROR, "AML: no rw on /sys/module/amlvideodri/parameters/freerun_mode");
-      permissions_ok = 0;
-    }
     if (!SysfsUtils::HasRW("/sys/class/video/freerun_mode"))
     {
       CLog::Log(LOGERROR, "AML: no rw on /sys/class/video/freerun_mode");
